content/guides/access-control.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Enable and configure access control
+weight: 1
+description: A tutorial for creating Role-based Access Control in Upbound
+---
+
+{{< hint "important" >}}
+This feature is in alpha and requires Spaces `v1.3.0`.
+{{< /hint >}}
+
+This guide introduces role-based access control (RBAC) in Upbound. RBAC allows you to apply privileges to your Upbound objects and control planes. Organizations using RBAC can manage user roles and privileges to determine the level of access a user can have within their role.
+
+RBAC allows organization administrators to:
+
+- build and assign custom roles
+- have a centralized set of policies to ensure appropriate access
+- enhance compliance workflows
+
+This guide highlights three RBAC workflows you might use in Upbound:
+
+- Kubernetes cluster management
+    -
+- Spaces/IAM
+    -
+
```

Comment:
I think it would be nice to use pictures similar to this one from the document:
<img width="630" alt="image" src="https://github.com/upbound/docs/assets/5442475/2b88d031-17b3-437b-9e2e-9ef13c451b21">

maybe you have your own graphics style externally in the docs site, you could re-make this graphic then 👍